File: 소프티어/A+B.py
import sys

# 표준 입력을 'input'이라는 이름으로 사용.
input = sys.stdin.readline

# 내장 함수인 input()과는 다른 input()
T = int(input())

for i in range(T):
    A, B = map(int, input().split())

    print("Case #%d:"%(i+1), (A+B)) # %d == %변수

File: 소프티어/A+B.js
var fs = require('fs');
var input = fs.readFileSync(0).toString().trim().split('\n');

const test = Number(input[0]);

for (var i = 1; i < input.length; i++) {
    var value = input[i].split(' ');
    console.log(`Case #${i}: ${Number(value[0]) + Number(value[1])}`);
}